diff --git a/node_modules/react-server-dom-webpack/esm/react-server-dom-webpack-node-loader.production.js b/node_modules/react-server-dom-webpack/esm/react-server-dom-webpack-node-loader.production.js
index d6013b8..3b893a1 100644
--- a/node_modules/react-server-dom-webpack/esm/react-server-dom-webpack-node-loader.production.js
+++ b/node_modules/react-server-dom-webpack/esm/react-server-dom-webpack-node-loader.production.js
@@ -471,7 +471,11 @@ async function transformModuleIfNeeded(source, url, loader) {
     const sourceMapString = typeof sourceMapResult.source === 'string' ? sourceMapResult.source :
     // $FlowFixMe
     sourceMapResult.source.toString('utf8');
-    sourceMap = JSON.parse(sourceMapString);
+    try {
+      sourceMap = JSON.parse(sourceMapString);
+    } catch (e) {
+
+    }
 
     // Strip the source mapping comment. We'll re-add it below if needed.
     source = source.slice(0, sourceMappingStart) + '\n'.repeat(sourceMappingLines) + source.slice(sourceMappingEnd);
